### ISSUE TITLE
Unable to run

### ISSUE THREAD

**Issue by teicors** [hSaria/ChromaTerm--#36]
I just downloaded the source from Github and compiled the source; launching 
./ct
I receive the error 
sh: 1: exec: session: not found

My environment is Ubuntu 18.04

I'm putting the out of ./configure to better identify the issue:

 ./configure 

checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether make sets $(MAKE)... yes
checking for library containing forkpty... -lutil
checking how to run the C preprocessor... gcc -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking pty.h usability... yes
checking pty.h presence... yes
checking for pty.h... yes
checking util.h usability... no
checking util.h presence... no
checking for util.h... no
checking for library containing pcre2_compile_8... -lpcre2-8
checking for pcre2.h... yes
configure: creating ./config.status
config.status: creating Makefile
config.status: creating config.h
config.status: config.h is unchanged

**Comment by hSaria** [comment]
Just as a work around while I spin up an Ubuntu VM, could you try the following:
```
./ct -e "/bin/bash"
```

**Comment by teicors** [comment]
... mmhh , the same error:

angelo@angelo-U36SG:~/Software/ChromaTerm--/src$ ./ct -e "/bin/bash"
sh: 1: exec: session: not found

**Comment by hSaria** [comment]
On your shell, what is the output of:
```
echo $SHELL
```

**Comment by teicors** [comment]
Here's the output:

```
angelo@angelo-U36SG:~/Software/ChromaTerm--/src$ echo $SHELL
/usr/bin/bash
angelo@angelo-U36SG:~/Software/ChromaTerm--/src$ ls -lart /usr/bin/bash
lrwxrwxrwx 1 root root 9 gen 10 08:12 /usr/bin/bash -> /bin/bash

```

**Comment by teicors** [comment]
I solved !
I removed the line 
`#run session bash`
coming from the original .chromatermrc of tunnelup

**Comment by hSaria** [comment]
Oh, I see.

Thanks for solving it :)

**Comment by hSaria** [comment]
You'll want to get v0.1.1 (just `git clone` the master). It fixes #35 which is quite an annoying bug.

**Comment by teicors** [comment]
I got it !!
